### ISSUE TITLE
Chrome seems to be broken

### ISSUE THREAD

**Issue by binarious** [damko/freedommarks-browser-webextension#13]
I have the extension installed in Firefox and Chrome with the same configurations. It does work in Firefox but doesn't in Chrome. The developer console shows constant redirects (302) to the target Nextcloud instance.

**Comment by damko** [comment]
yes, sorry about that. The bug is confirmed since a while but I had no time to look into it and won't have for a while. 
I accept PRs, just saying

**Comment by binarious** [comment]
Alright. Going to look into it.

**Comment by binarious** [comment]
@damko Do you have a short instruction on how to get this running locally in chrome? Just adding the `src` folder as an extension leads to: 

> Unrecognized manifest key 'applications'.

**Comment by damko** [comment]
you need to go in developer mode https://developer.chrome.com/extensions/getstarted

**Comment by binarious** [comment]
Debugged for some time. Copied the exact same request chrome sends and put that into a REST api client (Insomia). I don't get a 302 there. Also used `apache_request_headers` in the index.php on my nextcloud instance to output the headers - no difference either.

People from the Passman webextension have similar issues:
https://github.com/nextcloud/passman-webextension/issues/258

Will try some other things when I find more time.

**Comment by damko** [comment]
In fact, if it works in FF it should also work in Chrome. It's a weird behavior. Anyway, thanks for the effort

**Comment by marcelklehr** [comment]
@damko I've had the same issue in floccus, here's the fix: https://github.com/nextcloud/passman-webextension/pull/270

**Comment by binarious** [comment]
Thanks @marcelklehr. Created a PR based on this.
